dask/dataframe/io/parquet/fastparquet.py

```diff
@@ -81,7 +81,7 @@ class FastParquetEngine(Engine):
 
     @staticmethod
     def read_metadata(
-        fs, fs_token, paths, categories=None, index=None, gather_statistics=None
```

Comment:
I'm curious about the change to drop the `fs_token` argument.  Was it no longer necessary?

Reply:
I'm not sure if it *should* be used, but I removed it wasn't being used (and I had come accross a recommendation to remove it in an earlier code review)